engine/src/main/java/org/terasology/world/block/tiles/WorldAtlasImpl.java

```diff
@@ -250,9 +269,18 @@ private void createTextureAtlas(final Texture texture) {
     }
 
     private ByteBuffer[] createAtlasMipmaps(int numMipMaps, Color initialColor, List<BlockTile> tileImages, String screenshotName) {
+        return createAtlasMipmaps(numMipMaps, initialColor, tileImages, screenshotName, Lists.newArrayList());
+    }
+
+    private ByteBuffer[] createAtlasMipmaps(int numMipMaps, Color initialColor, List<BlockTile> tileImages, String screenshotName, List<BlockTile> alphaMaskTiles) {
         ByteBuffer[] data = new ByteBuffer[numMipMaps];
         for (int i = 0; i < numMipMaps; ++i) {
             BufferedImage image = generateAtlas(i, tileImages, initialColor);
+            if (alphaMaskTiles.size() > 0) {
+                BufferedImage alphaMask = generateAtlas(i, alphaMaskTiles, Color.BLACK);
+                storeGreyscaleMapIntoAlpha(image, alphaMask);
```

Comment:
Hmmm... so you are saying that because the calling method is generic the callED method should be generic even though it's used only to specifically store normals data and specular data together? ;)

Anyway, I looked at the rest of the WorldAtlasImpl code and it generated enough question marks orbiting my head already: one more is not going to make a difference.